### ISSUE TITLE
[A11y] Add accessibility label for the "..." disclosure button

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#24081]
The “…” disclosure button for bookmark cells that opens the context menu should have an accessibility label that reads out “More options”

There is discussion around usability testing for potentially improving this experience by hiding the “…” disclosure button from VoiceOver and replacing it’s access by utilizing VoiceOver actions (accessed via the Rotor)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-11032)

**Comment by data-sync-user** [comment]
➤ Alina Moldovan commented:

Verified as fixed using v9000(50315) and iPhone 16 Pro (18.2)

Confirming that with the current implementation the “….“  is announced as “More options“.

**Comment by data-sync-user** [comment]
➤ Adina Petridean commented:

Verified as fixed on Firefox 136.0 (50550) with iPhone 14 Pro Max (iOS 16.2).
